db/couchdb.js

```diff
@@ -52,19 +57,23 @@ const seedReviews = async () => {
            helpfulCount : faker.datatype.number(2000),
            abuseReported : faker.datatype.boolean()
         }
-        records.push(newReview);
+        newProduct.reviews.push(newReview);
     }
+    records.push(newProduct);
     productNum++;
   }
   await axios.post(`${reviewsUrl}`, {docs: records})
   .then(() => {
-    console.log('SUCCSEFULLY ADDED ' + productNum - 1 + ' RECORDS TO REVIEW DATABASE');
+    console.log('SUCCSEFULLY ADDED ' + (productNum - 1) + ' RECORDS TO REVIEW DATABASE');
   })
   .catch((error) => {
     console.log('ERROR LOADING REVIEWS DATABASE', error);
   })
     }
   }
+  module.exports.seedAverageReviews = seedAverageReviews;
+  module.exports.seedReviews = seedReviews;
 
   require('make-runnable');
```

Comment:
I may have to ask you what that does! Looking it up now and not getting it.

Reply:
This looks good, Jordan. For me instead of passing {docs: records} to axios post, I found that it is much easier to query the db when I  post  the generator functions directly like { _id : productNum.toString(),  helpfulCount : faker.datatype.number(2000), abuseReported : faker.datatype.boolean()}